lib/sycamore/sycamore/transforms/standardizer.py

```diff
@@ -31,9 +31,10 @@ def fixer(self, text: str) -> Union[str, Tuple[str, date]]:
         """
         pass
 
+    @abstractmethod
     def standardize(self, doc: Document, key_path: List[str]) -> Document:
         """
-        Applies the fixer method to a specific field in the document as defined by the key_path.
+        Abstract method applies the fixer method to a specific field in the document as defined by the key_path.
```

Comment:
wait why is this an abstract method now?

Reply:
standardize can have different implement for each child class depending on the required behavior, hence it is kept as Abstract method in the parent.